pom.xml

```diff
@@ -133,9 +133,10 @@
         	<plugin>
                 <groupId>org.glassfish.build</groupId>
                 <artifactId>spec-version-maven-plugin</artifactId>
-                <version>1.3</version>
+                <version>1.4</version>
                 <configuration>
                     <spec>
```

Comment:
See here how to configure the plugin for jakarta support: javaee/spec-version-maven-plugin#5